bundles/0.10.0/bundle.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM scratch
```

Comment:
We've been chatting about the possibility of not maintaining old versions of the bundles. As the repo itself is versioned we technically just need one copy that we could update all of the time. The change for that doesn't technically need to be done in this pr unless it makes sense to do that. Maybe we can chat about it?